support/gitlab-runners/values.yaml

```diff
@@ -75,7 +75,7 @@ runners:
     cpuLimit: 1000m
     memoryLimit: 4Gi
     cpuRequests: 500m
-    memoryRequests: 2Gi
+    memoryRequests: 3Gi
```

Comment:
I think you should increase the limit "memoryLimit" as well . memoryRequests is a "soft" limit and you can go over it and nothing would happen. If you go over the memoryLimit , then you'd get Out of Memory error. 

I recommend reading https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#how-pods-with-resource-limits-are-run

Reply:
Also open a new PR for this change only (new branch) do not change existing PR.

Reply:
You also need to apply the limits manually. Ther are instructions in https://github.com/vmware/versatile-data-kit/tree/main/support/gitlab-runners but ping me if you need help.